### ISSUE TITLE
Docker settings hangs indefinitely when I try to change where to save images

### ISSUE THREAD

**Issue by KayaLuken** [docker/for-win#13477]
1. go to settings
2. change location of where to save images to a different drive (eg drive D)
3. settings will hang indefinitely with the loading icon animating forever and the background greyed (docker creates empty folders in the new location but thats it), unless Docker engine is restarted, in which case the settings will uselessly revert the disk image save location to the previous unwanted default setting.

Moral of the story is Docker seems to need to use Drive C, and only Drive C at all costs, both for installation (no choice of where to install) and general usage.

**Comment by MeowKim** [comment]
Seems to be a bug in latest version. 
It also appears for-mac(https://github.com/docker/for-mac/issues/6803).

Temporary workaround is install previous version(4.16) and change disk image location. 
And then reinstall latest version(4.19).

**Comment by zorgoz** [comment]
> 

But... where is that setting in 4.16?

**Comment by Amitdedhia6** [comment]
exactly, cannot find any place where you can change the image location in version 4.16.0 and 4.16.3. I am using Docker Desktop for windows and WSL2 as base engine

**Comment by Amitdedhia6** [comment]
@zorgoz I followed steps at this [link ](https://www.kindacode.com/article/docker-desktop-change-images-containers-directory/#The_Steps_For_Windows_Users_with_WSL_2_Backend)Docker Desktop (for Windows) in v4.16.3. It worked well.

**Comment by zorgoz** [comment]
> @zorgoz I followed steps at this [link ](https://www.kindacode.com/article/docker-desktop-change-images-containers-directory/#The_Steps_For_Windows_Users_with_WSL_2_Backend)Docker Desktop (for Windows) in v4.16.3. It worked well.

Yes, I have found this as well, and actually worked.
